docs/installation/debian.md

```diff
@@ -12,113 +13,121 @@ parent = "smn_linux"
+ ## Prerequisites
 
-> **Note**:
-> Debian contains a much older KDE3/GNOME2 package called ``docker``, so the
-> package and the executable are called ``docker.io``.
+ Docker requires a 64-bit installation regardless of your Debian version.
+ Additionally, your kernel must be 3.10 at minimum. The latest 3.10 minor
+ version or a newer maintained version are also acceptable.
 
-### Installation
+ Kernels older than 3.10 lack some of the features required to run Docker
+ containers. These older versions are known to have bugs which cause data loss
+ and frequently panic under certain conditions.
 
-Make sure you enabled the `jessie-backports` repository, as stated above.
+ To check your current kernel version, open a terminal and use `uname -r` to
+ display your kernel version:
 
-To install the latest Debian package (may not be the latest Docker release):
+     $ uname -r
 
-    $ sudo apt-get update
-    $ sudo apt-get install docker.io
+### Update your apt repository
 
-To verify that everything has worked as expected:
+Docker's `apt` repository contains Docker 1.7.1 and higher. To set `apt` to use from the new repository:
 
-    $ sudo docker run --rm hello-world
+ 1. If you haven't already done so, log into your Ubuntu instance.
 
-This command downloads and runs the `hello-world` image in a container. When the
-container runs, it prints an informational message. Then, it exits.
+ 2. Open a terminal window.
 
-If you need to add an HTTP Proxy, set a different directory or partition for the
-Docker runtime files, or make other customizations, read our Systemd article to
-learn how to [customize your Systemd Docker daemon options](/articles/systemd/).
+ 3. Add the new `gpg` key.
 
-> **Note**:
-> If you want to enable memory and swap accounting see
-> [this](/installation/ubuntulinux/#adjust-memory-and-swap-accounting).
+         $ apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
 
-### Uninstallation
+ 4. Open the `/etc/apt/sources.list.d/docker.list` file in your favorite editor.
 
-To uninstall the Docker package:
+     If the file doesn't exist, create it.
 
-    $ sudo apt-get purge docker.io
+ 5. Remove any existing entries.
 
-To uninstall the Docker package and dependencies that are no longer needed:
+ 6. Add an entry for your Debian operating system.
 
-    $ sudo apt-get autoremove --purge docker.io
+     The possible entries are:
 
-The above commands will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
+       # Debian Wheezy
```

Comment:
Should all of these be backquoted? It looks funky.
<img width="736" alt="screen shot 2015-10-02 at 18 34 13" src="https://cloud.githubusercontent.com/assets/13337345/10260854/3bade3ae-6934-11e5-851d-74775ff624b7.png">